packages/containers/SignIn/SignInForm.js

```diff
@@ -71,7 +71,7 @@ const ButtonWrapper = styled.div`
   align-items: center;
   justify-content: center;
   margin-top: 30px;
-  margin-bottom: 60px;
+  margin-bottom: 30px;
```

Comment:
What about sign up page? I think there are the same styles. And if you will toggle this pages content will "jump".

Reply:
SignUp page has not any elements under button so there is no jump while toggling these pages